www/app/login/login.html

```diff
@@ -0,0 +1,15 @@
+<div class="backgroundInitialPage">
+<img src="../assets/images/smart_quote_logo.png" class="logo_smartquote">
+  <div class="list">
+    <label class="user_information_box item item-input">
+      <span class="input-label"> Correo </span>
+      <input type="text">
+    </label>
+    <label class="user_information_box item item-input">
+      <span class="input-label"> Contraseña </span>
+      <input type="password">
+    </label>
+  </div> 
+<a ui-sref="categories"><button class="button_login" > Iniciar </button></a>
+<a ui-sref="categories" class="button_sing_in"> Crear Cuenta </a> 
+</div>
```

Comment:
1. Minor detail, but indentation is important for understanding who is child of who. Indent `img` element properly.
2. Same for `a` elements
3. `img` is not well formed. You are not closing that element tag.
4. I'd expect email input is validated correctly.